packages/client/wallets/smart-wallet/package.json

```diff
@@ -22,13 +22,8 @@
     "dependencies": {
         "@crossmint/client-sdk-base": "workspace:*",
         "@crossmint/common-sdk-base": "workspace:*",
-        "@zerodev/ecdsa-validator": "5.3.1",
-        "@zerodev/passkey-validator": "5.4.0",
-        "@zerodev/sdk": "5.3.3",
-        "@zerodev/webauthn-key": "5.3.1",
-        "permissionless": "0.1.36",
-        "viem": "2.17.5",
-        "zod": "3.22.4"
+        "ox": "0.6.9",
+        "viem": "2.17.5"
```

Comment:
Idea for future work: remove `viem` and rewrite with `ox` entirely. This will cut the number of external deps in half, and will let customers to use any version of `viem` (or `ethers`)

Reply:
Our plan is that the interface we export is a Viem wallet compatible one, so I think we'll stick to viem long-term. What do you think?

Reply:
Can be `viem`-compatible w/o using `viem` at runtime. But yeah if that's the long-term plan, probably best to keep it.